data-raw/superheroes_supertbl.R

```diff
@@ -0,0 +1,6 @@
+redcap_uri <- Sys.getenv("REDCAP_URI")
+token <- Sys.getenv("SUPERHEROES_REDCAP_API")
+
+superheroes_supertbl <- read_redcap(redcap_uri, token)
+
+usethis::use_data(superheroes_supertbl, overwrite = TRUE)
```

Comment:
Just to keep track of the new folders, so we're introducing a `data-raw` folder (in `.Rbuildignore`) and a `data` folder (not in `.Rbuildignore`) that hold a script and a datafile, respectively.

May be something to do separately, but in the past I had put testing data under `/inst/testdata` (also in `.Rbuildignore`, with a `create_test_data.R` script) so we should consolidate these under your new folders.or under `/inst`.

Reply:
I think related to my comment above. If this was only internal testing data `/inst/testdata` would make sense.

I was following [this section](https://r-pkgs.org/data.html#sec-data-data) on adding exported data to a package and generated those new directories with `usethis::use_data_raw()` and `usethis::use_data()` described in there but if we have a way besides exporting this we can roll those directories back.

Reply:
If it's a process used in R packages and a part of the `usethis` framework then ignore me haha.